packages/cadl-vscode/src/extension.ts

```diff
@@ -1,4 +1,5 @@
-import { stat } from "fs/promises";
+import { resolveModule, ResolveModuleHost } from "@cadl-lang/compiler/module-resolver";
```

Comment:
I assume rollup is doing a good job of pulling in minimum here, but can you just double check diff of rolled-up extension?

Reply:
yeah that's why I had to do this, if I included the compiler completely it does some bad things even though it shouldn't 
<img width="678" alt="image" src="https://user-images.githubusercontent.com/1031227/196997078-f7109d58-7595-4a94-a169-d2540f7138a4.png">